lxd/storage/drivers/utils.go

```diff
@@ -226,6 +227,36 @@ func createSparseFile(filePath string, sizeBytes int64) error {
 	return nil
 }
 
+// ensureVolumeBlockPath creates or resizes the raw block file for a volume.
+func ensureVolumeBlockPath(vol Volume, path string) error {
+	blockSize := vol.config["size"]
+	if blockSize == "" {
+		blockSize = defaultBlockSize
+	}
+
+	blockSizeBytes, err := units.ParseByteSizeString(blockSize)
+	if err != nil {
+		return err
+	}
+
+	if shared.PathExists(path) {
+		_, err = shared.RunCommand("qemu-img", "resize", "-f", "raw", path, fmt.Sprintf("%d", blockSizeBytes))
+		if err != nil {
+			return fmt.Errorf("Failed resizing disk image %s to size %s: %v", path, blockSize, err)
+		}
+	} else {
+		// If rootBlockPath doesn't exist, then there has been no filler function
```

Comment:
@stgraber there's a left over mention of rootBlockPath here, so I guess this should change to just `path` as that is the new variable, or change the variable back to `rootBlockPath``

Reply:
fixed